SQF/dayz_code/stringtable.xml

```diff
@@ -10603,6 +10603,14 @@
 			<English>20Rnd. DMR mag.</English>
 			<Russian>20п. DMR маг.</Russian>
 		</Key>
```

Comment:
Please move all these new strings down into the dayz_epoch package since they aren't going to be in vanilla anytime soon.